app/api/business/brief/brief_edit_business.py

```diff
@@ -214,6 +215,18 @@ def edit_opportunity(user_id, brief_id, edits):
 
         errors.append(message)
 
+    if (
+        closing_date_was_edited(brief.closed_at.to_iso8601_string(), previous_data['closed_at']) and
+        not brief_blackout_validator.validate_closed_at_blackout(brief.closed_at.to_iso8601_string())
+    ):
+        blackoutDates = brief_blackout_validator.get_blackout_dates()
+        message = (
+            'The closing date cannot be between ' + blackoutDates['startDate'].strftime('%d %B') + ' and ' +
```

Comment:
`startDate` and `endDate` might be `None` based on what `get_blackout_dates` returns.  Do you need to handle that scenario when the values are referenced?

Reply:
If no blackout dates exist (and `get_blackout_dates()` returns `None`), the `validate_closed_at_blackout` function will always return true, so the scenario should not exist. But for completeness, I an add an error message if blackout dates aren't valid.